script/github-actions/eslint-annotation-format.js

```diff
@@ -0,0 +1,48 @@
+module.exports = function(results) {
+  const resultsArr = results || [];
+
+  const summary = resultsArr.reduce(
+    function(seq, current) {
+      current.messages.forEach(function(msg) {
+        const logMessage = {
+          filePath: current.filePath,
+          message: msg.message,
+          line: msg.line,
+          column: msg.column,
+        };
+
+        if (msg.severity === 1) {
+          logMessage.type = 'warning';
+          seq.warnings.push(logMessage);
+        }
+        if (msg.severity === 2) {
```

Comment:
I would make this `else if` so it doesn't bother checking this condition if the previous condition already evaluated as true.